charts/incubator/hyperswitch-helm/templates/secrets.yaml

```diff
@@ -21,7 +21,6 @@ data:
   ROUTER__SECRETS__KMS_ENCRYPTED_ADMIN_API_KEY: ""
   ROUTER__SECRETS__KMS_ENCRYPTED_JWT_SECRET: ""
   ROUTER__SECRETS__MASTER_ENC_KEY: ""
-  DB_PASS: "{{ .Values.db.password }}"
 kind: Secret
 metadata:
   name: hyperswitch-secrets
```

Comment:
Please add postgresql secret also
```
{{- if not .postgresql.enabled }}
apiVersion: v1
data:
  password: "ZGJwYXNzd29yZA=="
kind: Secret
metadata:
  name: "{{ .Release.Name }}-postgresql"
  namespace: {{ .Release.Namespace }}
type: Opaque
{{- end }}
```

Reply:
A Secret object is already created by bitnami PG sub chart, we just take it, instead recreating

Reply:
yes bitnami PG sub chart will create postgresql secret. but if the user choose to user external DB we will need this. so `if not .postgresql.enabled` we will add this secret